configs/inference/simple_ascii_tree.toml

```diff
@@ -0,0 +1,11 @@
+model_name = "deepseek-ai/DeepSeek-R1-Distill-Qwen-7B"
+dataset = "kalomaze/ascii-tree-mix-it1"
+batch_size = 16
+dp = 4
+rollout_path = "outputs"
+output_path = "data_rollout"
+max_model_len = 2048
+
+[sampling]
+temperature = 1.0
+n = 16
```

Comment:
any reason for having different sampling param that the default for this task ? Or was it just for debugging